docs/source/references.rst

```diff
@@ -1,6 +1,19 @@
 ==========
 References
 ==========
+If you find our work useful in your research, please consider citing:
+
+Hardware and Experiments
+~~~~~~~~~~~~~~~~~~~~~~~~
+Z. Wang, R. Spica and M. Schwager, “Game Theoretic Motion Planning for Multi-Robot Racing,” In Proc. of the International Symposium on Distributed Autonomous Robotics Systems (DARS 18), October, 2018. [ PDF ]
```

Comment:
Links to PDF?

Reply:
can ignore as this was leftover before fork to alex's